frontend/document/koptinterface.lua

```diff
@@ -240,7 +266,7 @@ function KoptInterface:getCachedContext(doc, pageno)
         --self:logReflowDuration(pageno, dur)
         local fullwidth, fullheight = kc:getPageDim()
         logger.dbg("reflowed page", pageno, "fullwidth:", fullwidth, "fullheight:", fullheight)
-        self.last_context_size = fullwidth * fullheight + 128 -- estimation
+        self.last_context_size = fullwidth * fullheight + 3072 -- estimation
```

Comment:
What/why this 3072 ? Feels a bit random for your "power-of-two OCD" :)

Reply:
~48 elements in the table, conservative estimation says one key, value pair takes two words, so, 48 * 32 * 2 ;).

Reply:
Might be fine to have that as comments ? So we don't wonder in a few years when reading that code again :)

Reply:
Added a generic comment about size estimations in CacheItem ;).